src/app/service/bounty/create/CreateNewBounty.ts

```diff
@@ -130,40 +131,74 @@ export default async (guildMember: GuildMember, params: BountyCreateNew, guildID
 			description: newBounty.description,
 			fields: [
 				{ name: 'HashId', value: listOfBountyIds[0], inline: false },
-				{ name: 'Reward', value: newBounty.reward.amount + ' ' + newBounty.reward.currency.toUpperCase(), inline: true },
+				{ name: 'Reward', value: newBounty.reward.amount + ' ' + newBounty.reward.currency, inline: true },
```

Comment:
Why remove the upper case from here?

Reply:
It was part of my hotfix PR, uppercase was removed because I fixed it earlier in the hierarchy where we create newBounty object. That way db has consistent uppercase stored and we don't need to worrying about formatting while displaying.